diff --git a/globals.d.ts b/globals.d.ts
index da499940ebbaa462edb8ce6c2dd2f1b67a19b925..392534f0c90f09e5b3845c56aea1a85e9b8573e1 100755
--- a/globals.d.ts
+++ b/globals.d.ts
@@ -69,11 +69,6 @@ declare var AbortController: {
     new(): AbortController;
 };
 
-declare var AbortSignal: {
-    prototype: AbortSignal;
-    new(): AbortSignal;
-    // TODO: Add abort() static
-};
 //#endregion borrowed
 
 //#region ArrayLike.at()
